src/lib/layout/Searchbar.tsx

```diff
@@ -131,6 +132,18 @@ const ResultItem = ({
   onClose,
 }: ResultItemProps) => {
   const route = getRouteOptions(type)?.pathname;
+  const isAccountAddress = type === "Account Address";
+  const displayValue = useMemo(() => {
+    if (isAccountAddress) {
+      return metadata.icns.address || metadata.initiaUsername.address || value;
+    }
+    return value;
+  }, [
+    isAccountAddress,
+    metadata.icns.address,
+    metadata.initiaUsername.address,
+    value,
+  ]);
```

Comment:
is this change necessary? i believe we can assign names to contract/SDK module account